kymatio/scattering3d/backend/numpy_backend.py

```diff
@@ -47,7 +27,7 @@ def modulus_rotation(x, module=None):
             which is covariant to 3D translations and rotations.
     """
     if module is None:
-        module = np.zeros_like(x)
+        module = np.zeros_like(x, np.float64)
```

Comment:
Why `float64`? This will cause problems if `x` is `float32`.

Reply:
@edouardoyallon aren't we assuming all numpy operations are in float64? i feel like it was something we discussed at the sprint.